.jenkins/Jenkinsfile

```diff
@@ -51,11 +48,12 @@ def nonSimulationTest() {
                 git url: 'https://github.com/Microsoft/openenclave.git'
             }
             /*
-            Use oetools-azure image from OE Jenkins Registry
+            Use oetools-azure Dockerfile from Openenclave repository
             Remove the installed az-dcap-client from the container
             Install az-dcap-client in the container from the deb package we previously built
             */
-            docker.image(OETOOLS_IMAGE).inside('-u root --privileged -v /dev/sgx:/dev/sgx') {
+            def oetoolsImage = docker.build("oetools-test", "-f openenclave/.jenkins/Dockerfile ./openenclave")
+            oetoolsImage.inside('-u root --privileged -v /dev/sgx:/dev/sgx') {
```

Comment:
I think we can change this to "--device /dev/sgx:/dev/sgx" at some point.  I'm looking back at the openenclave Jenkinsfile, and somehow we're using "--privileged" there as well. Hmm. 

At some point, I think we used "--device" docker option in openenclave, and now we aren't using it anymore.  I think we should move back to using the "--device" docker option, to expose less to a container.

Let's plan on changing this in a future issue, but for now this is fine!